src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java

```diff
@@ -147,16 +205,16 @@ private boolean isClientError(Exception e) {
                 e instanceof VerificationException;
     }
 
-    private RestChannelConsumer reportError(Exception e, RestStatus status) {
-        return sendResponse(new ErrorMessage(e, status.getStatus()).toString(), status);
+    private void sendResponse(final RestChannel channel, final String message, final RestStatus status) {
```

Comment:
is it possible to avoid these unnecessary change.

Reply:
the return type is changed, this is necessary.